server/models/pokemon.ts

```diff
@@ -0,0 +1,157 @@
+import { Model, RelationMappings, Transaction } from 'objection';
+import Item from './item';
+
+export enum PokemonType {
+  normal = 'normal',
+  grass = 'grass',
+  fire = 'fire',
+  water = 'water',
+  electric = 'electric',
+  psychic = 'psychic',
+  ghost = 'ghost',
+  dark = 'dark',
+  fairy = 'fairy',
+  rock = 'rock',
+  ground = 'ground',
+  steel = 'steel',
+  flying = 'flying',
+  fighting = 'fighting',
+  bug = 'bug',
+  ice = 'ice',
+  dragon = 'dragon',
+  poison = 'poison',
+}
+
+export interface IPokemonCreateFields {
+  id: string;
+  pokemonNumber: number;
+  name: string;
+  attack: number;
+  defense: number;
+  pokeType: PokemonType;
+  moves: string[];
+  imageUrl: string;
+}
+
+export interface IPokemonEditInput {
+  name: string;
+  attack: number;
+  defense: number;
+  pokeType: PokemonType;
+  moves: string[];
+  imageUrl: string;
+}
+
+export default class Pokemon extends Model {
+  // static modelPaths = [__dirname]
+  static pickJsonSchemaProperties = true;
+  static tableName = 'pokemon';
+
+  // tslint:disable-next-line: member-ordering
+  static jsonSchema = {
+    type: 'object',
+    properties: {
+      id: { type: 'string', format: 'uuid' },
+      pokemonNumber: { type: 'number' },
+      name: { type: 'string' },
+      attack: { type: 'number' },
+      defense: { type: 'number' },
+      pokeType: {
+        type: 'string',
+        enum: [
+          'normal',
+          'grass',
+          'fire',
+          'water',
+          'electric',
+          'psychic',
+          'ghost',
+          'dark',
+          'fairy',
+          'rock',
+          'ground',
+          'steel',
+          'flying',
+          'fighting',
+          'bug',
+          'ice',
+          'dragon',
+          'poison',
+        ],
+      },
+      moves: { type: 'array', items: { type: 'string' } },
+      imageUrl: { type: 'string' },
+      createdAt: { type: 'string' },
+      updatedAt: { type: 'string' },
+      deletedAt: { type: ['string', 'null'] },
+    },
+    required: ['pokemonNumber', 'name', 'attack', 'defense', 'pokeType', 'moves', 'imageUrl'],
+  };
+
+  // tslint:disable-next-line: member-ordering
+  static async create(pokemon: IPokemonCreateFields, txn: Transaction) {
+    return this.query(txn).insertAndFetch(pokemon);
+  }
+
+  static async getAll(txn: Transaction) {
+    return this.query(txn).orderBy('pokemonNumber');
+  }
+
+  static async get(pokeminId: string, txn: Transaction) {
```

Comment:
```suggestion
  static async get(pokemonId: string, txn: Transaction) {
```
also, don't forget return types